.github/workflows/build.yml

```diff
@@ -0,0 +1,40 @@
+name: Build
+on:
+  push:
+    branches:
+      - main
+  pull_request:
+
+jobs:
+  build:
+    strategy:
+      matrix:
+        go: [ "1.13","1.14","1.15", "1.16" ]
```

Comment:
Are you sure that's required? To me, it looks a bit too eager. It's minor versions. So testing 1.13 and 1.16 should be enough.

Reply:
Okay, let's test against 13 and 17